addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -1252,6 +1259,19 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
             const allowMultipleFiles = params.activeValue > 1;
             this.$target[0].toggleAttribute("multiple", allowMultipleFiles);
         }
+        if (params.attributeName === "requirementComparator") {
+            this.$target[0].dataset.customError = "";
+            this.$target[0].dataset.errorMessage = "";
+            this.$target[0].dataset.requirementCondition = "";
+            this.$target[0].dataset.requirementBetween = "";
+        }
+        if (params.attributeName === "requirementCondition" || params.attributeName === "requirementBetween") {
+            const comparator = this.$target[0].dataset.requirementComparator;
+            const condition = this.$target[0].dataset.requirementCondition;
+            const between = this.$target[0].dataset.requirementBetween;
+            const type = this.$target[0].dataset.type;
```

Comment:
Destructuring is kind of magic sometimes :)
```suggestion
            const { requirementComparator, requirementCondition, requirementBetween, type } = this.$target[0].dataset;
```